src/pages/settings/Subscription/SubscriptionDetails/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {View} from 'react-native';
+import {useOnyx} from 'react-native-onyx';
+import Icon from '@components/Icon';
+import * as Illustrations from '@components/Icon/Illustrations';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import type {OptionsPickerItem} from '@components/OptionsPicker';
+import OptionsPicker from '@components/OptionsPicker';
+import Section from '@components/Section';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function SubscriptionDetails() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const [privateSubscription] = useOnyx(ONYXKEYS.NVP_PRIVATE_SUBSCRIPTION);
+
+    const [selectedOption, setSelectedOption] = React.useState<string>(privateSubscription?.type ?? CONST.SUBSCRIPTION.TYPE.ANNUAL);
+    const [account] = useOnyx(ONYXKEYS.ACCOUNT);
+
+    const options: OptionsPickerItem[] = [
+        {
+            key: CONST.SUBSCRIPTION.TYPE.ANNUAL,
+            title: translate('subscription.details.annual'),
+            icon: Illustrations.SubscriptionAnnual,
+        },
+        {
+            key: CONST.SUBSCRIPTION.TYPE.PAYPERUSE,
+            title: translate('subscription.details.payPerUse'),
+            icon: Illustrations.SubscriptionPPU,
+        },
+    ];
```

Comment:
Looks like we can move it outside of the component to avoid re-creation of this array on each update.
```suggestion
    const options: OptionsPickerItem[] = [
        {
            key: CONST.SUBSCRIPTION.TYPE.ANNUAL,
            title: 'subscription.details.annual',
            icon: Illustrations.SubscriptionAnnual,
        },
        {
            key: CONST.SUBSCRIPTION.TYPE.PAYPERUSE,
            title: 'subscription.details.payPerUse',
            icon: Illustrations.SubscriptionPPU,
        },
    ];
```

Reply:
Nice improvement, thanks!